### ISSUE TITLE
Error when modifying new order

### ISSUE THREAD

**Issue by nkasimova** [erdewit/ib_insync#509]
python 3.9.6
ib_insync 0.9.71

I place order and want to update the price through some time. But when I do it - I have TypeError.
However, if I comment the lines with placing order (since it already exists in the exchange) and then modify price - everything will work.

How to reproduce:
1. run the code below
2. Comment placing order
3. Run again

```
from ib_insync import *
import dataclasses

util.logToConsole()

SYMBOL = "EURUSD"

ib = IB()
ib.connect('localhost', 7497, clientId=2)

contract = Forex(SYMBOL)
ib.qualifyContracts(contract)

# comment from here
order = LimitOrder('BUY', 20_000, 0.5)
trade = ib.placeOrder(contract, order)
print(f"Limit order was placed: {order}")

print("Sleep 5 seconds")
ib.sleep(5)
# till here

print("Update price to 5.00001")

trades = ib.openTrades()
print(f"trades: {trades}")
for t in trades:
    if t.contract.localSymbol.replace('.', '') == SYMBOL:
        updatedOrder = dataclasses.replace(t.order)
        updatedOrder.lmtPrice = round(0.5 + 0.00001, 5)
        print(f"Updated order: {updatedOrder}")
        new_trade = ib.placeOrder(t.contract, updatedOrder)
        print(f"new_trade: {new_trade}")
        break

print(f"Sleep 10 seconds")
ib.sleep(10)

trades = ib.openTrades()
print(f"trades: {trades}")
for trade in trades:
    if trade.contract.localSymbol.replace('.', '') == SYMBOL:
        print("Cancel order....")
        ib.cancelOrder(trade.order)

ib.run()
```

My logs after placing new order (after step 1):
    2022-09-21 14:42:35,799 ib_insync.client INFO API connection ready
    2022-09-21 14:42:36,102 ib_insync.ib INFO Synchronization complete
    2022-09-21 14:42:36,477 ib_insync.ib INFO placeOrder: New order Trade(contract=Forex('EURUSD', conId=12087792, exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=LimitOrder(orderId=50, clientId=2, action='BUY', totalQuantity=20000, lmtPrice=0.5), orderStatus=OrderStatus(orderId=50, status='PendingSubmit', filled=0.0, remaining=0.0, avgFillPrice=0.0, permId=0, parentId=0, lastFillPrice=0.0, clientId=0, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 42, 36, 477510, tzinfo=datetime.timezone.utc), status='PendingSubmit', message='', errorCode=0)], advancedError='')
    Limit order was placed: LimitOrder(orderId=50, clientId=2, action='BUY', totalQuantity=20000, lmtPrice=0.5)
    Sleep 5 seconds
    2022-09-21 14:42:36,785 ib_insync.wrapper INFO orderStatus: Trade(contract=Forex('EURUSD', conId=12087792, exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=LimitOrder(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, lmtPrice=0.5, auxPrice=0.0), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 42, 36, 477510, tzinfo=datetime.timezone.utc), status='PendingSubmit', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 42, 36, 785197, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0)], advancedError='')
    Update price to 5.00001
    trades: [Trade(contract=Forex('EURUSD', conId=12087792, exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=LimitOrder(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, lmtPrice=0.5, auxPrice=0.0), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 42, 36, 477510, tzinfo=datetime.timezone.utc), status='PendingSubmit', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 42, 36, 785197, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0)], advancedError='')]
    Traceback (most recent call last):
      File "/Users/nadezhda/Documents/Finance/IBTestOrders/test_to_github.py", line 214, in <module>
        updatedOrder = dataclasses.replace(t.order)
      File "/usr/local/Cellar/python@3.9/3.9.6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/dataclasses.py", line 1284, in replace
        return obj.__class__(**changes)
      File "/usr/local/lib/python3.9/site-packages/ib_insync/order.py", line 179, in __init__
        Order.__init__(
    TypeError: ib_insync.order.__create_fn__.<locals>.__init__() got multiple values for keyword argument 'orderType'
    2022-09-21 14:42:41,500 ib_insync.ib INFO Disconnecting from localhost:7497, 383 B sent in 10 messages, 26.6 kB received in 392 messages, session time 5.71 s.
    2022-09-21 14:42:41,500 ib_insync.client INFO Disconnecting

Logs after re-running, not placing order - just modifying the exist one  (after step 3)
    2022-09-21 14:43:55,705 ib_insync.client INFO API connection ready
    2022-09-21 14:43:56,006 ib_insync.ib INFO Synchronization complete
    Update price to 5.00001
    trades: [Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.5, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0)], advancedError='')]
    Updated order: Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.50001, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True)
    2022-09-21 14:43:56,275 ib_insync.ib INFO placeOrder: Modify order Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.5, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0)], advancedError='')
    new_trade: Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.5, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0)], advancedError='')
    Sleep 10 seconds
    2022-09-21 14:43:56,563 ib_insync.wrapper INFO orderStatus: Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.50001, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='PreSubmitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 562604, tzinfo=datetime.timezone.utc), status='PreSubmitted', message='', errorCode=0)], advancedError='')
    2022-09-21 14:43:56,580 ib_insync.wrapper INFO orderStatus: Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.50001, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 562604, tzinfo=datetime.timezone.utc), status='PreSubmitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 579982, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0)], advancedError='')
    trades: [Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.50001, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='Submitted', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 562604, tzinfo=datetime.timezone.utc), status='PreSubmitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 579982, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0)], advancedError='')]
    Cancel order....
    2022-09-21 14:44:06,278 ib_insync.ib INFO cancelOrder: Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.50001, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='PendingCancel', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 562604, tzinfo=datetime.timezone.utc), status='PreSubmitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 579982, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 44, 6, 278286, tzinfo=datetime.timezone.utc), status='PendingCancel', message='', errorCode=0)], advancedError='')
    2022-09-21 14:44:06,570 ib_insync.wrapper INFO orderStatus: Trade(contract=Forex('EURUSD', conId=12087792, right='?', exchange='IDEALPRO', localSymbol='EUR.USD', tradingClass='EUR.USD'), order=Order(orderId=50, clientId=2, permId=2038089191, action='BUY', totalQuantity=20000.0, orderType='LMT', lmtPrice=0.50001, auxPrice=0.0, tif='DAY', ocaType=3, displaySize=2147483647, trailStopPrice=1.5, volatilityType=0, deltaNeutralOrderType='None', referencePriceType=0, account='DU5896513', clearingIntent='IB', adjustedOrderType='None', cashQty=0.0, dontUseAutoPriceForHedge=True), orderStatus=OrderStatus(orderId=50, status='Cancelled', filled=0.0, remaining=20000.0, avgFillPrice=0.0, permId=2038089191, parentId=0, lastFillPrice=0.0, clientId=2, whyHeld='', mktCapPrice=0.0), fills=[], log=[TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 55, 704059, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 275141, tzinfo=datetime.timezone.utc), status='Submitted', message='Modify', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 562604, tzinfo=datetime.timezone.utc), status='PreSubmitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 43, 56, 579982, tzinfo=datetime.timezone.utc), status='Submitted', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 44, 6, 278286, tzinfo=datetime.timezone.utc), status='PendingCancel', message='', errorCode=0), TradeLogEntry(time=datetime.datetime(2022, 9, 21, 11, 44, 6, 569723, tzinfo=datetime.timezone.utc), status='Cancelled', message='', errorCode=0)], advancedError='')
    2022-09-21 14:44:06,570 ib_insync.wrapper INFO Warning 202, reqId 50: Order Canceled - reason:

**Comment by erdewit** [comment]
```
updatedOrder = dataclasses.replace(t.order)
File "/usr/local/Cellar/python@3.9/3.9.6/Frameworks/Python.framework/Versions/3.9/lib/python3.9/dataclasses.py", line 1284, in replace
return obj.class(**changes)
File "/usr/local/lib/python3.9/site-packages/ib_insync/order.py", line 179, in init
Order.init(
TypeError: ib_insync.order.create_fn..init() got multiple values for keyword argument 'orderType'
```

This happens in your code.

**Comment by mattsta** [comment]
> TypeError: ib_insync.order.create_fn..init() got multiple values for keyword argument 'orderType'

so the dataclass copy is actually not the correct as I thought previously (trying to copy a non-dataclass using `dataclasses.replace()` causes that error).

`Order` is a dataclass, but the other types like `LimitOrder` etc *are not* dataclasses, so only `Order` itself can use `dataclasses.replace(order)` while the others can't be copied as easily.

For my own system I changed to _only_ using `Order` dataclass and never using `LimitOrder`, `MarketOrder`, `StopLimitOrder` wrappers 🤷‍♂️ (and dataclasses don't inherit cleanly due to parameter ordering conflicts, so there's not a great way to have top-level `Order` wrappers — in my own platform I just made an abstract `Order` wrapper that knows how to create more concrete order types like: `order = CustomOrder(action="BUY", qty=300, lmt=2.33).limitOrder()` then it returns `ib_insync.Order()` populated with the correct values/parameters).
